step-function/get-files/src/main.py

```diff
@@ -17,7 +17,7 @@ def get_expiration_time(key):
     s3_object = S3_CLIENT.get_object(Bucket=environ['BUCKET'], Key=key)
     expiration_string = s3_object['Expiration'].split('"')[1]
     expiration_datetime = datetime.strptime(expiration_string, '%a, %d %b %Y %H:%M:%S %Z')
```

Comment:
Haven't found a way to get a timezone-aware datetime out of this.  `expiration_string` is of the form `Fri, 10 Jul 2020 00:00:00 GMT`.  `GMT` is a timezone name, (%Z), not a timezone offset (%z).  See notes for %z/%Z at https://docs.python.org/3/library/datetime.html#technical-detail